modules/ossm-about-service-mesh-custom-resource-definitions.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+// * install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-service-mesh-custom-resource-definitions_{context}"]
+= About Service Mesh Custom Resource Definitions
+
+Installing the {SMProductName} Operator also installs Custom Resource Definitions (CRD) that administrators can use to configure {istio} for {SMProductShortName} installations. The {olm-first} installs two categories of CRDs: {sail-operator} CRDs and {istio} CRDs.
+
+{sail-operator} CRDs define custom resources for installing and maintaining the {istio} components required to operate a service mesh. These custom resources belong to the `sailoperator.io` API group and include the `Istio`, `IstioRevision`, `IstioCNI`, and `ZTunnel` resource kinds. For more information on how to configure these resources, see the `sailoperator.io` link:https://github.com/istio-ecosystem/sail-operator/blob/main/docs/api-reference/sailoperator.io.md[API reference] documentation.
```

Comment:
Checking here that you have been cleared to link to the `istio-ecosystem` repo?

Reply:
Yes. I guess I should have commented about that. I discussed this Kathryn and Kathryn suggested linking for now. This is a TP doc set (also highly unusual, and Kathryn is aware of that as well.) After we move to XML we can revisit this...the GA date is still a few months out and likely could slip. We do have some customers using the TP product to provide dev feedback, so that's why the program obtained approval to publish TP doc.